vehicles/src/modules/permit/application.controller.ts

```diff
@@ -107,7 +118,13 @@ export class ApplicationController {
       UserAuthGroup.CV_CLIENT === currentUser.orbcUserAuthGroup
         ? currentUser.userGUID
         : null;
-    return this.applicationService.findAllApplications({
+    const applicationStatus: ApplicationStatus[] =
```

Comment:
Refactor to util file to return a list based on currentUser auth group

Reply:
Done.